api/cirrus_ci_service.proto

```diff
@@ -158,6 +158,7 @@ message PollRequest {
   repeated int64 old_running_tasks = 2 [deprecated = true];
   map<string, double> resources_in_use = 3;
   repeated string running_tasks = 4;
+  StandbyInstanceInformation availableStandbyInstanceInformation = 5;
```

Comment:
We should probably use snake case here too, to be consistent with the rest of the file.